gen-apidocs/generators/html.go

```diff
@@ -602,8 +602,8 @@ func (h *HTMLWriter) generateHTML(navContent string) {
 
 	/*
 	Physical locations of files referenced:
-	reference-docs/gen-apidocs/generators/static/jquery.scrollTo.min.js
-	reference-docs/gen-apidocs/generators/static/scroll.js
+	reference-docs/gen-apidocs/static/js/jquery.scrollTo.min.js
+	reference-docs/gen-apidocs/static/js/scroll.js
```

Comment:
@tengqm , a couple more nits, then ready to merge.
- The comment on line 608 is misleading as the API reference does not use bootstrap-*.min.js.
- Line 621, pull out js/bootstrap-4.3.1.min.js. This file is not in the original code or Makefile?
- I will rebuild the ref using the changes once this goes in.

Reply:
We do use the bootstrap JS implicitly, for toggling the embedded examples.

Reply:
Where is bootstrap*.js stored?

Reply:
https://github.com/kubernetes/website/blob/master/static/js/bootstrap-4.3.1.min.js

Reply:
OK